src/index.tsx

```diff
@@ -12,10 +12,12 @@ import AppConfig from './domain/app/AppConfig';
 initI18next();
 Modal.setAppElement('#root');
 
-if (import.meta.env.VITE_ENVIRONMENT === 'production') {
+console.warn('Environment prod: ', import.meta.env.PROD);
+
+if (import.meta.env.PROD) {
```

Comment:
This is the important change! 👍 

Now it's right, since the `import.meta.env.PROD` in VITE is same as `process.env.NODE_ENV === 'production'` in many other Node frameworks. The `import.meta.env.PROD` should be `true` everytime the app package is built and bundled (like the NODE_ENV === 'production' in similar conditions). 

The Sentry SDK should not be initialized in development environment (meaning developers local computers). There is no option to disable the Sentry after it's initialized.

References:
- https://vite.dev/guide/env-and-mode#node-env-and-modes
- https://v2.vitejs.dev/guide/env-and-mode.html#env-variables-and-modes